src/components/SignerPanel/ConfirmMsgSign.js

```diff
@@ -0,0 +1,191 @@
+import React from 'react'
```

Comment:
This has very similar logic to the `ActionPathsContent` component and parts can be abstracted into shared component logic